tests/safeds/ml/nn/layers/test_forward_layer.py

```diff
@@ -177,3 +177,37 @@ def test_should_assert_that_different_forward_layers_have_different_hash(
 )
 def test_should_assert_that_layer_size_is_greater_than_normal_object(layer: ForwardLayer) -> None:
     assert sys.getsizeof(layer) > sys.getsizeof(object())
+
+
+@pytest.mark.parametrize(
+    ("activation_function", "expected_activation_function"),
+    [
+        ("sigmoid", nn.Sigmoid),
+        ("relu", nn.ReLU),
+        ("softmax", nn.Softmax),
+        ("none", None),
+    ],
+    ids=["sigmoid", "relu", "softmax", "none"],
+)
+def test_should_set_activation_function(activation_function: str, expected_activation_function: type | None) -> None:
+    forward_layer: ForwardLayer = ForwardLayer(1)
+    match activation_function:
+        case "sigmoid":
+            forward_layer = ForwardLayer(neuron_count=1, overwrite_activation_function="sigmoid")
+        case "relu":
+            forward_layer = ForwardLayer(neuron_count=1, overwrite_activation_function="relu")
+        case "softmax":
+            forward_layer = ForwardLayer(neuron_count=1, overwrite_activation_function="softmax")
+        case "none":
+            forward_layer = ForwardLayer(neuron_count=1, overwrite_activation_function="none")
```

Comment:
Is there a reason for the switch statement? Otherwise Simplify to
```suggestion
        forward_layer = ForwardLayer(neuron_count=1, overwrite_activation_function=activation_function)
```

Reply:
yes otherwise mypy throws an error, because a string is not a valid literal

Reply:
Try changing the type of the activation_function parameter from str to Literal["sigmoid","relu", ....]

Reply:
changed